plugins/http_plugin/http_plugin.cpp

```diff
@@ -465,7 +520,7 @@ namespace eosio {
             auto next_ptr = std::make_shared<url_handler>(std::move(next));
             return [this, priority, next_ptr=std::move(next_ptr)]( detail::abstract_conn_ptr conn, string r, string b, url_response_callback then ) mutable {
                auto tracked_b = make_in_flight<string>(std::move(b), *this);
-               if (!conn->verify_max_bytes_in_flight()) {
+               if (!conn->verify_max_bytes_in_flight()  || !conn->verify_max_requests_in_flight()) {
```

Comment:
I don't think its necessary to call verify_max_requests_in_flight here, we establish this request's presence in the in-flight set on line 622 & 623